### ISSUE TITLE
QR codes don't scan on LineageOS

### ISSUE THREAD

**Issue by BostonZcash** [zcash/zcash-android-wallet-sdk#135]
**What is the bug?**
In attempting to scan z-address QR codes (no URI or payment requests embedded), app does not recognize any such code (regardless of whitespace, etc.). Rather, it behaves as if no QR code is being presented.

**Additional context**
LineageOS 16.0 & ECC Wallet 1.0.0-alpha29. No Google Play Services (openGApps) have sullied this device.

**Solution**
I ran into the same issue on initial builds of [adityapk00/zqwandroid](https://github.com/adityapk00/zqwandroid), later corrected. As I recall, it was due to relying on a Google library to perform decoding/parsing, which is not included on base LineageOS images.

**Comment by gmale** [comment]
The firebase machine learning scanner is really nice and integrates fairly easily with the new CameraX APIs. I'd like to keep it. In theory, it should work, even if the device doesn't have

**Comment by gmale** [comment]
The firebase machine learning scanner is really nice and integrates fairly easily with the new CameraX APIs. I'd like to keep it, if possible.

In theory, it should work, even if the device doesn't have play services. I anticipated that this situation might come up so I built the QR scanning separate from the SDK, as an independent module that can be easily replaced in the app.

Closing this issue for the SDK and re-opening it on the ECC wallet, instead:
https://github.com/zcash/zcash-android-wallet/issues/165
